docs/matrix-org/privacy_notice.md

```diff
@@ -174,6 +176,12 @@ Access control settings, requests to remove messages by "redactions" or remove p
 
 The {{ policy_homeserver }} homeserver provides a range of integrations in the form of Widgets (miniature web applications accessed as part of a Matrix Client) and Bots (automated participants in rooms). Bots and Widgets currently have access to all the messages and files in any room in which they participate, although we are adding a more sophisticated access control system.
 
+### Transfers of your Data
+
```

Comment:
UK/EU? matrix.org runs from both

Reply:
I'll switch this to EU for now - it's good 'til Oct 31st at least.